From 8be2c08141f0275e9da005bd143771953e982a59 Mon Sep 17 00:00:00 2001
From: lorenz <looooo@users.noreply.github.com>
Date: Sun, 3 May 2020 07:47:27 +0200
Subject: [PATCH] partdesign: fix failing tapered hole test

The given parameters return an invalid shape. This fails with occt7.4 but doesn't with occt7.3. If the angle is 45 degree the cone is self-intersecting as Hole.Depth > Hole.Diameter/2. Changing the Hole.TaperedAngle to 60 degree solves this issue.
---
 src/Mod/PartDesign/PartDesignTests/TestHole.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Mod/PartDesign/PartDesignTests/TestHole.py b/src/Mod/PartDesign/PartDesignTests/TestHole.py
index d84afd69ca..cc838f4a3b 100644
--- a/src/Mod/PartDesign/PartDesignTests/TestHole.py
+++ b/src/Mod/PartDesign/PartDesignTests/TestHole.py
@@ -68,7 +68,7 @@ class TestHole(unittest.TestCase):
     def testTaperedHole(self):
         self.Hole.Diameter = 6
         self.Hole.Depth = 5
-        self.Hole.TaperedAngle = 45
+        self.Hole.TaperedAngle = 60
         self.Hole.ThreadType = 0
         self.Hole.HoleCutType = 0
         self.Hole.DepthType = 0
-- 
2.30.1

